src/pwacompat.js

```diff
@@ -139,13 +139,25 @@ function unused() {
     }
   }
 
+  /**
+   * @param {object} icon
+   * @param {string} icon.sizes
+   */
+  function largestSize(icon) {
+    const sizes = icon.sizes.split(' ').map((size) => {
+      if (size === 'any') return Infinity;
+      return parseInt(size, 10);
+    });
+    return Math.max(...sizes);
```

Comment:
If `parseInt` returns `NaN`, then `Math.max` will return it first.

To be fair, the `icons.sort` call below had the same problem. But can we filter out anything falsey, or return zero?

`parseInt(size, 10) || 0` should do it.